app-misc/python-dice/python-dice-3.1.0.ebuild

```diff
@@ -0,0 +1,35 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} )
+
+inherit distutils-r1
+
+DESCRIPTION="Dice parsing and evaluation library"
+HOMEPAGE="https://github.com/borntyping/python-dice"
+SRC_URI="https://github.com/borntyping/python-dice/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="test"
```

Comment:
Your `distutils_enable_tests` call takes care of the testing boilerplate; you can drop DEPEND and IUSE but it should be moved to just after the dependency declarations.